lib/python/gladevcp/tooledit_widget.py

```diff
@@ -308,7 +308,7 @@ def reload(self,widget):
                                 print(_("Tooledit widget float error"))
                         else:
                             try:
-                                array[offset]= locale.format_string("%10.4f", float(word.lstrip(i)))
+                                array[offset]= locale.delocalize(locale.format_string("%10.4f", float(word.lstrip(i))))
```

Comment:
locale.delocalize(locale.format_string()) is a bit ... hmm ... twofold.
First add the proper locale format and then remove it? 
A better style would be to use simply the string format function.

Reply:
Can string format solve problems with , and . ?
In CS localization it works.

Reply:
Yes, but we need decimal point "comma" akceptable for DE and CS and other localization.

Reply:
I agree that folks have dumped particular platforms for less of a reason than for not allowing them to regularly use their number pad.